nens_auth_client/users.py

```diff
@@ -14,13 +15,8 @@
 User = get_user_model()
 
 
-def _extract_provider_name(claims):
-    """Return provider name from claim and `None` if not found"""
-    # Also used by backends.py
-    try:
-        return claims["identities"][0]["providerName"]
-    except (KeyError, IndexError):
-        return
+def contains_including_wildcard(elem: str, set_):
+    return "*" in set_ or elem in set_
```

Comment:
Doet wat 'ie moet doen, maar de naamgeving bevalt me niet helemaal.

- Qua naam verwacht ik eerder nog wat wildcard-matching zoals `*.pietje.nl`.
- "lijst" contains "item => `elem` en `set_` verwacht ik omgekeerd.
- Als er een "*" in een lijst staat verwacht ik dat het ook het enige item is en dan is `"*" in set_` wat raar.

```suggestion
def found_or_wildcard(element: str, allowed_elements: Iterableofzo):
    return "*" in allowed_elements or element in allowed_elements
```

Vanuit "element" gezien is het dan "is 'ie gevonden of is 'ie toegestaan omdat het een wildcard is", dat loopt wat lekkerder.